js/AddTodolistItem.js

```diff
@@ -0,0 +1,155 @@
+import {
+    getUserIdAndDeleteTodolist,
+    getUserIdAndItemComplete,
+    getUserIdAndModifyTodolist,
+    getUserIdAndAddTodolist
+} from './Server.js';
+
+function addTodolistItem(value, item) {
+    checkServerItemComplete(value,item);
+    clickEraseButton();
+    clickCheckboxButton();
+    clickLabel();
+    //strong태그 안에 있는 count 값 적용시키기
+    document.querySelector('.todo-count > strong').innerText = document.querySelector('.todo-list').childElementCount;
+
+}
+function clickSet() {
+//전체보기 버튼 클릭시 모든 내용 display
+    document.querySelector('.all').addEventListener('click', onAllFilterHandler);
+
+//해야할 일 버튼 클릭시 class가 completed인 것들은 display가 none이 되도록
+    document.querySelector('.active').addEventListener('click', onActiveFilterHandler);
+
+//완료한 일 클릭시 class가 active인 것들은 display가 none이 되도록
+    document.querySelector('.completed').addEventListener('click', onCompletedFilterHandler);
+}
+
+function onAllFilterHandler(){
+    document.querySelectorAll('.todo-list > li').forEach(x => x.style.display = '');
+    /* 아래코드도 가능
+   let child = document.querySelector('.todo-list')
+   child.childNodes.forEach(x=> x.style.display = '');
+     */
+}
+
+function onActiveFilterHandler(){
+    document.querySelectorAll('.todo-list > li').forEach(x => x.classList.contains('active') == true ? x.style.display = '' : x.style.display = 'none');
```

Comment:
```suggestion
    document.querySelectorAll('.todo-list > li')
                     .forEach(({classList, style}) => { 
                          classList.contains('active') ? style.display = '' : style.display = 'none'
                     });
```
코드의 한줄이 길면 가독성이 떨어지니 . 을 기준으로 줄바꿈을 해주면 좋겠네요